website/docs/r/evidently_launch.html.markdown

```diff
@@ -0,0 +1,349 @@
+    steps {
+      group_weights = {
+        "Variation1" = 0
+      }
+      start_time = "2024-01-07 01:43:59+00:00"
+    }
+  }
+}
+```
+
+### With randomization_salt
+
+```terraform
+resource "aws_evidently_launch" "example" {
+  name               = "example"
+  project            = aws_evidently_project.example.name
+  randomization_salt = "example randomization salt"
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation1"
+    variation = "Variation1"
+  }
+
+  scheduled_splits_config {
+    steps {
+      group_weights = {
+        "Variation1" = 0
+      }
+      start_time = "2024-01-07 01:43:59+00:00"
+    }
+  }
+}
+```
+
+### With multiple steps
+
+```terraform
+resource "aws_evidently_launch" "example" {
+  name    = "example"
+  project = aws_evidently_project.example.name
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation1"
+    variation = "Variation1"
+  }
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation2"
+    variation = "Variation2"
+  }
+
+  scheduled_splits_config {
+    steps {
+      group_weights = {
+        "Variation1" = 15
+        "Variation2" = 10
+      }
+      start_time = "2024-01-07 01:43:59+00:00"
+    }
+
+    steps {
+      group_weights = {
+        "Variation1" = 20
+        "Variation2" = 25
+      }
+      start_time = "2024-01-08 01:43:59+00:00"
+    }
+  }
+}
+```
+
+### With segment overrides
+
+```terraform
+resource "aws_evidently_launch" "example" {
+  name    = "example"
+  project = aws_evidently_project.example.name
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation1"
+    variation = "Variation1"
+  }
+
+  groups {
+    feature   = aws_evidently_feature.example.name
+    name      = "Variation2"
+    variation = "Variation2"
+  }
+
+  scheduled_splits_config {
+    steps {
+      group_weights = {
+        "Variation1" = 0
+        "Variation2" = 0
+      }
+
+      segment_overrides {
+        evaluation_order = 1
+        segment          = aws_evidently_segment.example.name
+
+        weights = {
+          "Variation2" = 10000
+        }
+      }
+
+      segment_overrides {
+        evaluation_order = 2
+        segment          = aws_evidently_segment.example.name
+
+        weights = {
+          "Variation1" = 40000
+          "Variation2" = 30000
+        }
+      }
+
+      start_time = "2024-01-08 01:43:59+00:00"
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `description` - (Optional) Specifies the description of the launch.
+* `groups` - (Required) One or up to five blocks that contain the feature and variations that are to be used for the launch. [Detailed below](#groups).
+* `metric_monitors` - (Optional) One or up to three blocks that define the metrics that will be used to monitor the launch performance. [Detailed below](#metric_monitors).
```

Comment:
```suggestion
* `metric_monitors` - (Optional) One or up to three blocks that define the metrics that will be used to monitor the launch performance. [Detailed below](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/evidently_launch#metric_monitors).
```

Reply:
Following the Kendra docs https://github.com/hashicorp/terraform-provider-aws/blob/fcb0d1ed516f6036c145db8f1ea76709c2298382/website/docs/r/kendra_experience.html.markdown?plain=1#L49 full length URLs are not required for linking, pretty cool stuff